@navikt/core/css/darkside/button.darkside.css

```diff
@@ -173,10 +170,7 @@
   }
 
   &:active {
-    --__axc-button-border-color: transparent;
-
-    color: var(--ax-text-contrast);
```

Comment:
it seems like so many changes, I'm guessing they are fine? (i don't seem to find any differences so far tabbing between this chromatic & a local one from main).

Here it seems active state is weaker on already weaker surfaces, and probably the intent right? 💭 (it makes sense to me to not have to jump to super high contrast for the active state if the current is very low contrast)

Reply:
These changes are only affecting `:active`-state, so that we no longer "flip" the color, but rather keep it on moderate for active-state